litgpt/scripts/download.py

```diff
@@ -37,16 +38,18 @@ def download_from_hub(
         model_name: The existing config name to use for this repo_id. This is useful to download alternative weights of
             existing architectures.
     """
-    print("repo_id:", repo_id)
+    options = [f"{config['hf_config']['org']}/{config['hf_config']['name']}" for config in configs]
 
     if repo_id == "list":
-        from litgpt.config import configs
-
-        options = [f"{config['hf_config']['org']}/{config['hf_config']['name']}" for config in configs]
         print("Please specify --repo_id <repo_id>. Available values:")
         print("\n".join(sorted(options, key=lambda x: x.lower())))
         return
 
+    if repo_id not in options:
```

Comment:
With this disabled, it is now no longer possible to download finetuned models from other repos. The `--model_name` argument that was useful for this previously now has no use anymore. Do I see this right?

Can we either remove the argument or disable this prevention if model_name is provided?

Reply:
Follow-up in #1620